jobs/e2e_node/perf-image-config.yaml

```diff
@@ -0,0 +1,9 @@
+---
+images:
+  cosbeta-resource1:
```

Comment:
For parity with image-config.yaml and image-config-serial.yaml
```suggestion
  cos-stable1:
```

Reply:
Fixed